google/cloud/opentelemetry/quickstart/README.md

```diff
@@ -0,0 +1,201 @@
+# HOWTO: Google Cloud C++ Client Library + Cloud Trace integration.
+
+This directory contains a small example showing how to enable OpenTelemetry
+tracing for a Google Cloud Storage (GCS) client. The traces are collected and
+sent to Cloud Trace.
+
+This quickstart assumes the reader is familiar with how to build and use the GCS
+client. If not, see the [GCS quickstart](/google/cloud/storage/quickstart).
+
+The quickstart also assumes the reader is familiar with how to build the Cloud
+Trace C++ client library. If not, see the
+[Cloud Trace quickstart](/google/cloud/trace/quickstart). (Note that the Cloud
+Trace Exporter requires dependencies on gRPC and Protobuf, which the GCS client
+does not).
+
+## The Quickstart
+
+### What it does
+
+The quickstart installs a Cloud Trace [exporter]. It creates a GCS client, with
+tracing enabled. The client makes some routine calls, each of which are traced.
+Upon exit of the quickstart, the collected traces are sent to Cloud Trace.
+
+### Viewing the trace in Cloud Trace
+
+To find the traces created by this quickstart, we can follow the documentation
+to [Find and view traces]. We can open the Cloud Trace UI in the Google Cloud
+project we supplied to the quickstart. We can search using the filter
+`SpanName:storage::Client` to find any spans created by the GCS client.
+
+![google_cloud_cpp_opentelemetry_quickstart](https://user-images.githubusercontent.com/23088558/235268195-c19bf903-7358-4a0f-ac72-438b5286463b.png)
```

Comment:
This will not work if we ever want to move this to the reference documents.  Maybe we should commit this as an asset:

https://github.com/GoogleCloudPlatform/cpp-samples/tree/main/getting-started/assets